docs/sources/enterprise/configuration.md

```diff
@@ -0,0 +1,187 @@
++++
+title = "Enterprise configuration"
+description = "Enterprise configuration documentation"
+keywords = ["grafana", "configuration", "documentation", "enterprise"]
+type = "docs"
+[menu.docs]
+name = "Enterprise configuration"
+identifier = "enterprise-config"
+parent = "enterprise"
+weight = 300
++++
+
+# Grafana Enterprise configuration
+
+This page describes Grafana Enterprise-specific configuration options that you can specify in a `.ini` configuration file or using environment variables. Refer to [Configuration]({{< relref "../installation/configuration.md" >}}) for more information about available configuration options.
+
+## [white_labeling]
+
+### app_title
+
+Set to your company name to override application title.
+
+### login_logo
+
+Set to complete url to override login logo.
+
+### login_background
+
+Set to complete css background expression to override login background. Example: 
+
+```bash
+[white_labeling]
+login_background = url(http://www.bhmpics.com/wallpapers/starfield-1920x1080.jpg)
+```
+
+### menu_logo
+
+Set to complete url to override menu logo.
+
+### fav_icon
+
+Set to complete url to override fav icon (icon shown in browser tab).
+
+### apple_touch_icon
+
+Set to complete url to override apple/ios icon.
+
+### footer_links
+
+List the links id's to use here, Grafana will look for matching links configurations the link id's should be space separated and contain no whitespace.
+
+## [meta_analytics]
+
+### max_file_age
+
+Max age for data files before they get deleted.
+
+### max_data_directory_size
+
+Max size in megabytes of the data files directory before files gets deleted.
+
+### data_path
+
+The directory where events will be stored in.
+
+## [analytics.summaries]
+
+### buffer_write_interval
+
+Interval for writing dashboard usage stats buffer to database.
+
+### buffer_write_timeout
+
+Timeout for writing dashboard usage stats buffer to database.
+
+### rollup_interval
+
+Interval for trying to rollup per dashboard usage summary only rolled up at most once per day.
```

Comment:
This sentence confuses me. I think we have at least two sentences in here. Please clarify.

Reply:
Also, 

roll up = verb
rollup = noun

Perform a rollup on the summary.
Try to roll up the summary.

Please correct throughout.